_posts/minecraft/java/2024-07-14-server-properties.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Configuring Java Minecraft Server: Key Attributes of server.properties"
+category: 
+    - Java
+    - Getting-started
```

Comment:
The post only shows up in the Getting-started category, you should only add one category